system-addon/content-src/styles/_icons.scss

```diff
@@ -18,39 +18,39 @@
   }
 
   &.icon-bookmark-added {
-   background-image: url('chrome://browser/skin/bookmark.svg');
+    background-image: url('chrome://browser/skin/bookmark.svg');
  }
```

Comment:
Some of these close braces `}` need to be indented 1 more space to match the opening brace line.

Reply:
Changes updated in https://github.com/mozilla/activity-stream/pull/3751/commits/c882ccfe133da0b6051088f242adcb4eb273ebe0